### PR TITLE
Fixed import issues when used as an external library with Python 3

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

File: elifetools/parseJATS.py [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 from collections import OrderedDict
-from six import iteritems
 
+from bs4 import BeautifulSoup
 from slugify import slugify
 
-from utils import unicode_value
+import elifetools.rawJATS as raw_parser
+import elifetools.json_rewrite
+from elifetools.utils import *
```
Comment (giorgiosironi) [inline elifetools/parseJATS.py]: `import *` is suspicious

Comment (giorgiosironi) [reply elifetools/parseJATS.py]: it was already there

Comment (seanwiseman) [reply elifetools/parseJATS.py]: Yes, I am holding off on the refactor until the imports are sorted.

**Comment by gnott** [comment]
Could it be possible to shorten the usage with something like

```
import elifetools.json_rewrite as json_rewrite
```

then all the usages of it can drop the ``elifetools.`` part?

**Comment by seanwiseman** [comment]
I need to look into it further, but aliasing using `as` gives different results in that very use case because of the circular imports between `json_rewrite` and `parseJATS`. I will give some more details once I have finished and you can take a look. The use case where this all comes from is when `bot-lax-adaptor` imported `elifetools` as a dependancy, obviously, before that, all `elifetools` tests were passing and it was only when I tried importing it that these relative imports became an issue.

I will try and shorten any I can once it all works, but this one was intentional. Its also possible a small refactor would assist this, but I'm holding off to see if it can be done without first.

Due to the relative import changes in `Python 3` you either need to be explicit (using `elifetools.`) or move them into sub packages so they can resolve the names correctly.

**Comment by seanwiseman** [comment]
I have refactored the import in `parseJATS` so its now `import elifetools.json_rewrite as json_rewrite` but the one in`json_rewrite` has to stay as `import elifetools.parseJATS` otherwise, you get an import error when using it as a library.

**Comment by gnott** [comment]
I can see some minor changes I could do to make it so ``json_rewrite.py`` does not need to import ``parseJATS.py`` if it would help in the integration with other librraries.

**Comment by seanwiseman** [comment]
@gnott  If you are happy with this now that your refactor(s) are in I am happy to merge.

**Comment by gnott** [comment]
It is compatible with the tests and other libraries, looks great!

**Comment by seanwiseman** [comment]
@giorgiosironi  are you happy for me to merge this?

**Comment by giorgiosironi** [comment]
No problem for merging, I didn't have particular concerns.

**Pull request merged**